test/skip-subtypes/generated/BodyComplex/README.md

```diff
@@ -1,88 +1,83 @@
 ## An isomorphic javascript sdk for - AutoRestComplexTestService
 
-This package contains an isomorphic SDK for AutoRestComplexTestService.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for AutoRestComplexTestService.

```

Comment:
@ramya-rao-a , here is an updated readme file. @joheredi how can I get a display name for a package? I currently just use a normalized package name.